src/identification.js

```diff
@@ -0,0 +1,109 @@
+"use strict";
+
+const fs = require("fs");
+const net = require("net");
+const colors = require("colors/safe");
+const Helper = require("./helper");
+
+class Identification {
+	constructor(startedCallback) {
+		this.connectionId = 0;
+		this.connections = new Map();
+
+		if (typeof Helper.config.oidentd === "string") {
+			this.oidentdFile = Helper.expandHome(Helper.config.oidentd);
+			log.info(`Oidentd file: ${colors.green(this.oidentdFile)}`);
+
+			this.Refresh();
+		}
+
+		if (Helper.config.identd.enable) {
+			if (this.oidentdFile) {
+				log.warn("Using both identd and oidentd at the same time, this is most likely not intended.");
+			}
+
+			var server = net.createServer(this.ServerConnection.bind(this));
+			server.listen({
+				port: Helper.config.identd.port || 113,
+				host: Helper.config.bind || Helper.config.host,
+			}, () => {
+				var address = server.address();
+				log.info(`Identd server available on ${colors.green(address.address + ":" + address.port)}`);
+
+				startedCallback();
+			});
+		} else {
+			startedCallback();
+		}
+	}
+
+	ServerConnection(socket) {
+		socket.on("data", (data) => {
+			this.RespondToIdent(socket, data);
+			socket.end();
+		});
+	}
+
+	RespondToIdent(socket, data) {
+		data = data.toString();
+		data = data.split(",");
```

Comment:
`data = data.toString().split(",");` instead? Or does that hide something else like a missing `if` or whatever?